frontend/src/app/sessions/user-sessions-wrapper/create-session/provision-workspace/provision-workspace.component.html

```diff
@@ -0,0 +1,33 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+<div class="wrapper">
+  <h2 id="title">Provision workspace</h2>
+  <mat-card id="readonly-card">
+    <mat-card-content>
+      <p>Load the models in a training environment.</p>
+
+      <form [formGroup]="provisionForm" (ngSubmit)="requestSessions()">
+        <fieldset *ngIf="projectType !== 'training'">
+          <mat-slide-toggle formControlName="persistentWorkspace">
+            Use a persistent workspace</mat-slide-toggle
+          >
+        </fieldset>
+
+        <fieldset class="mt-4">
+          <button
+            matInput
+            mat-flat-button
+            color="primary"
+            type="submit"
+            [disabled]="!projectSlug || !models || sessionsRequested"
+          >
+            Start this training
+            <mat-icon>keyboard_arrow_right</mat-icon>
+          </button>
+        </fieldset>
```

Comment:
```suggestion
        <button
          matInput
          mat-flat-button
          class="w-full"
          color="primary"
          type="submit"
          [disabled]="!projectSlug || !models || sessionsRequested"
        >
          Start this training
          <mat-icon>keyboard_arrow_right</mat-icon>
        </button>
```
Here we can completely remove the `fieldset`, again to fix the sonar cloud issue (we then need to add the `w-full` to handle the difference layouting.